pages/linux/resolvconf.md

```diff
@@ -0,0 +1,29 @@
+# resolvconf
+
+> Manage nameserver information.
+> Acts as an intermediary between rograms that supply nameserver information and applications that use this information.
+> More information: <https://manpages.ubuntu.com/manpages/resolvconf.8.html>.
```

Comment:
```suggestion
> More information: <https://manned.org/resolvconf.8>.
```

We use manned.org for vendor-neutral manpage documentation—See #5143.

Reply:
This link https://manned.org/resolvconf.8 documents `operesolve` implementation and I am documenting Debian's implementation. Shall I change it or is there any way to add all the variations on the same page?